features/networking/operator.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Operator related networking scenarios
+  
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-22704
+  @admin
+  Scenario: The clusteroperator should be able to reflect the network operator version corresponding to the OCP version
+    Given I have a project
```

Comment:
you don't need to create a project

Reply:
Agree, Peter. We don't need that step, guess i am accustomed to perform all operations in separate project unless a specific one is required.